quickwit/quickwit-config/src/quickwit_config/mod.rs

```diff
@@ -40,20 +42,20 @@ pub const DEFAULT_QW_CONFIG_PATH: &str = "config/quickwit.yaml";
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct IndexerConfig {
-    #[doc(hidden)]
-    #[serde(default = "IndexerConfig::default_enable_opentelemetry_otlp_service")]
-    pub enable_opentelemetry_otlp_service: bool,
     #[serde(default = "IndexerConfig::default_split_store_max_num_bytes")]
     pub split_store_max_num_bytes: Byte,
     #[serde(default = "IndexerConfig::default_split_store_max_num_splits")]
     pub split_store_max_num_splits: usize,
     #[serde(default = "IndexerConfig::default_max_concurrent_split_uploads")]
     pub max_concurrent_split_uploads: usize,
+    #[doc(hidden)]
```

Comment:
should we remove `#[doc(hidden)]` here?

Reply:
Yes, I'm also looking for a more user-friendly name.